tools/pontoon/resources.md

```diff
@@ -0,0 +1,95 @@
+# Resources for translating in Pontoon
+
+## Machinery
+
+Pontoon features built-in access to different translation tools and repositories. These resources are available in the workspace and on a standalone web page.
+
+When translating, double click any machinery entry to auto-fill the editing space with the machinery translation.
+
+### History tab
+
+The history tab shows each of the suggestions and translations that have been submitted for the current string.
+
+In the tab, the number of history entry is visible alongside the title, `HISTORY`. The history entries are locate below the tab.
+
+![History tab](/assets/images/pontoon/resources/history.png)
+
+At the left of the entry, the name of the translator is visible in gray and the date of submission is visible in green.
+
+To the right of the entry, a circle icon indicates the status of the entry (see [string type]()). By clicking the trashcan icon, a translator can remove the entry from the string's history. And below the icons, the translation or suggestion is visible.
+
+### Machinery tab
+
+The Machinery tab shows possible translations from a variety of sources. These sources include Pontoon's internal translation memory, external translation memories, machine translation engines, and other external sources.
+
+In the tab, the number of machinery entries is visible alongside the title, `MACHINERY`.
+
+![Machinery tab](/assets/images/pontoon/resources/machinery.png)
+
+Below the tab, there is a machinery search bar. A translator can enter text into the search bar to search for any strings in the machinery resources that may be similar. The search does not need to be related tot he current project string.
+
+At the left of the entry, a diff view of the current source string and the source string from the machinery resource is visible. Red-highlighted text indicates text that is present in the current string that is not in the machinery string. Green-highlighted text indicates text that is present in the machinery string that is not present in the current project string.
+
+To the right of the entry, the number in green shows the percent match between the machinery string and the current string. The higher the percentage, the more likely the machinery is to be useful. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right of the source indicates the number of entries for the entry string in the source.
```

Comment:
Having a hard time making this sentence concise yet clear. The next commit contains "The green superscript to the right indicates the number of matches for the entry in the source."